src/main/java/com/webperside/courseerpbackend/controller/StudentController.java

```diff
@@ -0,0 +1,24 @@
+package com.webperside.courseerpbackend.controller;
+
+import com.webperside.courseerpbackend.models.base.BaseResponse;
+import com.webperside.courseerpbackend.models.payload.student.StudentPayload;
+import com.webperside.courseerpbackend.services.student.StudentService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/v1/students")
+@RequiredArgsConstructor
+public class StudentController {
+    private final StudentService studentService;
+
+    @PostMapping("/add")
```

Comment:
if we follow REST rules, we don't need "/add" endpoint. Because if you are using /v1/students POST method it means you will add students via this endpoint
keep only @PostMapping